packages/pharos-site/static/guidelines/textarea.docs.mdx

```diff
@@ -5,22 +5,22 @@ import BestPractices from '../../src/components/statics/BestPractices.tsx';
   isHeader
   title="Textarea"
   storyBookType="forms"
-  description="A text area allows users to enter long, free-form text that can be on multiple lines."
+  description="A text area allows users to enter long, freeform text that can be on multiple lines."
 ></PageSection>
 
 <PageSection title="Usage">
   <p>
     Textarea allows users to enter a sizeable amount of free-form text, like leaving feedback,
-    adding a description, creating a note and more. If an input requires users to write longer forms
-    of text, use a textarea instead of a single text input field.
+    adding a description, creating a note, and more. If an input requires users to write longer
+    forms of text, use a textarea instead of a single text-input field.
   </p>
 </PageSection>
 
 <PageSection title="Best practices">
   <BestPractices
     Do={
       <ul>
-        <li>Labels should clearly indicate input expectations of the user</li>
+        <li>Labels should clearly indicates input expectations of the user</li>
```

Comment:
This should remain `indicate`—"Labels...indicate"

Alternatively, "Labels" should change to "The label"—"The label...indicates"